includes/Modules/Analytics.php

```diff
@@ -126,6 +127,25 @@ function () {
 			},
 			0
 		);
+
+		// Add the timezone data for users provisioning new accounts.
+		if ( ! $this->is_connected() ) {
+			add_filter(
+				'googlesitekit_admin_data',
+				function( $admin_data ) {
+					$selected_zone = get_option( 'timezone_string' );
+
+					$timezones     = new Timezones();
+					$timezone_data = $timezones->get_timezone_data();
+
+					$admin_data['timezones'] = $timezone_data;
+					$admin_data['timezone']  = $selected_zone;
+
+					$admin_data['errorcode'] = $this->context->input()->filter( INPUT_GET, 'error_code', FILTER_SANITIZE_STRING );
```

Comment:
capture and later display an error code or message, potentially passed back by the callback.